modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,328 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cluster-cloud-controller-config_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM) config map
```

Comment:
Should we remove RH reference from the CCM name as it's not provided/developed by Red Hat exclusively?
@MaysaMacedo @dulek @stephenfin please correct me if I'm wrong
```suggestion
= The OpenStack Cloud Controller Manager (CCM) config map
```

Reply:
works for me to have only OpenStack

Reply:
@stephenfin The idea is that I'm required to respect RH branding practices whenever I write about our products, so the PR is littered with "RHOSP" (`{rh-openstack}`). It's muscle memory. :) 

If OpenStack CCM is not subject to RH branding guidelines, I'm free to use "OpenStack" in place of "RHOSP."

My assumption here is that OpenStack CCM is usable outside of Red Hat's OpenStack solution. If that's not the case, let me know.